daffodil-japi/src/main/scala/org/apache/daffodil/japi/infoset/Infoset.scala

```diff
@@ -244,7 +246,7 @@ class XMLTextInfosetOutputter private (outputter: SXMLTextInfosetOutputter)
    *               insert indentation and newlines where it will not affect the
    *               content of the XML.
    */
-  def this(os: java.io.OutputStream, pretty: Boolean) = this(new SXMLTextInfosetOutputter(os, pretty))
+  def this(os: java.io.OutputStream, pretty: Boolean, xmlOutputStyle: XMLOutputStyle) = this(new SXMLTextInfosetOutputter(os, pretty, xmlOutputStyle))
```

Comment:
I think this should be a new constructor, so we have both a 2-arg and 3-arg constructor, and each one calls the appropriate SXMLTextInfosetOutputter constructor. This would allow for backwards compatibility. Ideally users of the current XMLTextInfosetOutputter (Including existing API tests) shouldn't be required to add the 3 parameter.